ClientCore/ProgramConstants.cs

```diff
@@ -26,7 +26,7 @@ public static class ProgramConstants
 
         public const string QRES_EXECUTABLE = "qres.dat";
 
-        public const string CNCNET_PROTOCOL_REVISION = "R10";
+        public const string CNCNET_PROTOCOL_REVISION = "R11";
```

Comment:
Why?

Reply:
Its live right now, and not bumping it would mean other players would still see the game and join?

Reply:
Well, it isn't the intended use case of protocol version but I guess that also works.